static/js/public/snap-details/screenshots.js

```diff
@@ -13,7 +13,12 @@ export default function initScreenshots(screenshotsId) {
     const url = event.target.src;
 
     if (url) {
-      lightbox.openLightbox(url, images);
+      if (prev.offsetWidth === 0 && prev.offsetHeight === 0) {
```

Comment:
Feels like a strange way to detect 'small screen'. Why is `prev.offsetWidth` used for that?

Reply:
It's a way to check visibility of an element, I should probably add a comment.